docs/date.md

```diff
@@ -1,96 +1,398 @@
+A `Temporal.Date` can be converted into a `Temporal.DateTime` by combining it with a `Temporal.Time` using the `withTime()` method.
+
+`Temporal.YearMonth` and `Temporal.MonthDay` carry less information than `Temporal.Date` and should be used when complete information is not required.
 
 ## Constructor
 
 ### **new Temporal.Date**(_year_: number, _month_: number, _day_: number, _disambiguation_: 'constrain' | 'balance' | 'reject' = 'constrain') : Temporal.Date
 
-Creates a new `Date` object that represents a calendar date
+**Parameters:**
+- `year` (number): A year, ranging between &minus;271821 and 275760 inclusive.
+- `month` (number): A month, ranging between 1 and 12 inclusive.
+- `day` (number): A day of the month, ranging between 1 and 31 inclusive.
+- `disambiguation` (optional string): How to deal with out-of-range values of the other parameters.
+  Allowed values are `constrain`, `balance`, and `reject`.
+  The default is `constrain`.
+
+**Returns:** a new `Temporal.Date` object.
+
+Use this constructor if you have the correct parameters for the date already as individual number values, or you need the disambiguation behaviour.
+Otherwise, `Temporal.Date.from()`, which accepts more kinds of input, is probably more convenient.
+
+The `disambiguation` parameter works as follows:
+- In `constrain` mode (the default), any out-of-range values are clamped to the nearest in-range value.
+- In `balance` mode, any out-of-range values are resolved by balancing them with the next highest unit.
+- In `reject` mode, the presence of out-of-range values will cause the constructor to throw a `RangeError`.
+
+Usage examples:
+```javascript
+// Pi day in 2020
+date = new Temporal.Date(2020, 3, 14)  // => 2020-03-14
+
+// Different disambiguation modes
+date = new Temporal.Date(2001, 13, 1, 'constrain')  // => 2001-12-01
+date = new Temporal.Date(2001, -1, 1, 'constrain')  // => 2001-01-01
+date = new Temporal.Date(2001, 13, 1, 'balance')  // => 2002-01-01
+date = new Temporal.Date(2001, -1, 1, 'balance')  // => 2000-11-01
+date = new Temporal.Date(2001, 13, 1, 'reject')  // throws
+date = new Temporal.Date(2001, -1, 1, 'reject')  // throws
+```
 
 ## Static methods
 
 ### Temporal.Date.**from**(_thing_: string | object) : Temporal.Date
 
-### Temporal.Date.**compare**(_one_: Temporal.Date, _two_: Temporal.Date) : number
+**Parameters:**
+- `thing` (string or object): The value representing the desired date.
+
+**Returns:** a new `Temporal.Date` object (or the same object if `thing` was a `Temporal.Date` object.)
+
+This static method creates a new `Temporal.Date` object from another value.
+If the value is a string, it must be in ISO 8601 format.
+If the string designates a time zone, the time zone will be ignored.
+If the value is another `Temporal.Date` object, the same object is returned.
+If the value is any other object, it must have `year`, `month`, and `day` properties, and a `Temporal.Date` will be constructed from them.
```

Comment:
This is not precisely correct.  The fields in .from will be passed verbatim to the default calendar, or to a custom calendar if it is provided in the option bag.  For example, `Temporal.Date.from({ era: "reiwa", year: 1, month: 1, day: 1, calendar: "japanese" })` is a valid thing to do; in this case, the options bag has an extra field.

Reply:
(later)